benchmark/csrc/sampler/mapper.cpp

```diff
@@ -0,0 +1,85 @@
+#include <algorithm>
+#include <random>
+#include <type_traits>
+#include <vector>
+
+#include <benchmark/benchmark.h>
+
+#include "pyg_lib/csrc/sampler/cpu/mapper.h"
+
+template <typename node_t, typename scalar_t>
+class BenchmarkMapperCreationAndInsertion : public benchmark::Fixture {
+  static_assert(std::is_integral<node_t>::value &&
+                    std::is_integral<scalar_t>::value,
+                "Integral type required for both node_t and scalar_t");
+
+ protected:
+  void SetUp(const benchmark::State& state) override {
+    static std::random_device rd;
```

Comment:
Not really familiar with this. Is it deterministic?

Reply:
Would it make sense to make this deterministic such that different runs are comparable? Not a blocker though.

Reply:
Yes, just did it.